README.md

```diff
@@ -121,3 +123,60 @@ In the RAUC hawkBit updater's configuration file it's called "gateway_token".
 Although gateway token is very handy during development or testing, it's
 recommended to use this token with care because it can be used to
 authenticate any device.
+
+
+Authentication via Certificates
+-------------------------------
+
+As can be seen in the system configuration settings of hawkBit, there is a
+third option to authenticate the targets. An "Allow targets to authenticate via
+a certificate authenticated by a reverse proxy" option. To use this
+authentication method a TLS reverse proxy server needs to be set up.
+The client and reverse proxy server perform a "SSL-handshake" that means the
+client validates the server certificate of the reverse proxy server with its
```

Comment:
The server also validates the client's certificate? I.e. we are performing "mutual authentication"?

Reply:
Yes, the client request has to contain the client certificate, which the server has to verify.